package/webpackDevServerConfig.js

```
@@ -0,0 +1,70 @@
+const snakeToCamelCase = require("./utils/snakeToCamelCase")
+const shakapackerDevServerYamlConfig = require("./dev_server")
+const { outputPath: contentBase, publicPath } = require("./config")
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)